3DAmsterdam/Assets/Amsterdam3D/Scripts/BAG-API/TabManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TabManager : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+}
```

Comment:
Deze class is leeg

Reply:
Ik heb hem verwijderd, bedankt voor het melden :)